examples/pxScene2d/src/pxWayland.cpp

```diff
@@ -569,45 +524,29 @@ rtError pxWayland::startRemoteObjectLocator()
 rtError pxWayland::connectToRemoteObject()
 {
   rtError errorCode = RT_FAIL;
-#ifdef ENABLE_PX_WAYLAND_RPC
-  int findTime = 0;
 
-  while (findTime < MAX_FIND_REMOTE_TIMEOUT_IN_MS && mClientPID != -1)
+#ifdef ENABLE_PX_WAYLAND_RPC
+  rtLogInfo("Attempting to find remote object %s", mRemoteObjectName.cString());
+  errorCode = rtRemoteLocateObject(mRemoteObjectName.cString(), mRemoteObject, 1000);
```

Comment:
I believe I've seen some long child app startups when search was taking over 1 second, it's safer to set it to some higher value (maybe 5 sec)
Thanks!